bobtemplates/plone/viewlet/browser/viewlets.py.example.bob

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+from plone.app.layout.viewlets import ViewletBase
+# -*- Extra imports go here -*-
+
```

Comment:
add a basic viewlet class here including a update method with some simple example code.

Reply:
I've also done this on a different branch and I'll push it here when done with the rest of the changes.